model_zoo/models/alexnet/model_alexnet.prototext

```diff
@@ -12,7 +12,7 @@ model {
   ###################################################
 
   objective_function {
-    cross_entropy {}
+    layer_term { layer: "cross_entropy_eval" }
```

Comment:
Maybe I just didn't see it in the code review, but how is the layer term used now?

Reply:
The evaluation layer was introduced in #276:
```
objective_function {
  layer_term {
    layer: "name_of_an_evaluation_layer"
    scale_factor: 1.0 // defaults to 1.0
  }
}
```
In the future, I'd like to add some logic to the model class so it can automatically insert evaluation layers where they're needed rather than forcing the practitioner to add them.